### DISCUSSION TITLE
Error GGML_ASSERT: whisper.cpp/ggml.c:3159: ggml_can_repeat_rows(b, a) when running stream.wasm

### DISCUSSION THREAD

**Discussion by andreineamtu** [ggml-org/whisper.cpp#1512] [Q&A]
Hi,

I'm trying to run the stream.wasm example and after I solved the SharedArrayBuffer error and CORS problems I now get this error when starting the stream:
Error GGML_ASSERT: whisper.cpp/ggml.c:3159: ggml_can_repeat_rows(b, a)

![Screenshot 2023-11-17 at 9 02 47 PM](https://github.com/ggerganov/whisper.cpp/assets/1506130/18362f5d-895a-47c4-b97a-61474542e8ca)

And is not transcribing  anything afterwards.
If anyone had this problem and solved it, it would be much appreciated.

**Comment by ggerganov** [comment]
This was fixed yesterday - pull latest `master`
